Applications/Games/Subnautica/Steam/script.js

```diff
@@ -0,0 +1,32 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Verbs", "vcrun2013"]);
+include(["Engines", "Wine", "Verbs", "vcrun2008"]);
+include(["Engines", "Wine", "Verbs", "d3dx9_42"]);
+include(["Engines", "Wine", "Verbs", "xact"]);
+include(["Engines", "Wine", "Verbs", "corefonts"]);
+
+new SteamScript()
+    .name("Subnautica")
+    .editor("Unknown Worlds")
```

Comment:
Shouldn't this be "Unknown Worlds Entertainment"?

Reply:
In their about page it's just "Unknown Worlds" but I will change that just in case.

Reply:
It's fine. I just saw that Steam uses "Unknown Worlds Entertainment".